### ISSUE TITLE
A couple of unit-tests are failing

### ISSUE THREAD

**Issue by garikello** [dmitigr/pgfe#11]
Running `make test` on **v2.0alpha3** tag yields an error :

> The following tests FAILED:
> 	  3 - misc-unit-mulf_valid (Failed)
> 	 10 - pgfe-unit-connection (Failed)
> 	 13 - pgfe-unit-hello_world (Failed)
> 

Platform: Arch Linux
Pq library version: 13.1-2

Configuration command line was:
`cmake -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql/ -DDMITIGR_CEFEIKA_BUILD_TESTS=TRUE`

Though I don't know whether I should have a live Postgres server running, I didn't see any option to set it.

**Comment by dmitigr** [comment]
Hello,

Thank you for the report! Yeah, some tests are broken now (and they are disabled until the refactoring) since Pgfe bumped up to 2.0alpha (and the API has been changed), although, `pgfe-unit-connection` and `pgfe-unit-hello_world` are working tests. In order to execute them you need to run Postgres locally and create the following objects:

```sql
CREATE USER pgfe_test ENCRYPTED PASSWORD 'pgfe_test';
CREATE DATABASE pgfe_test OWNER pgfe_test;
```

These objects are available in `test/pgfe/pgfe-deploy.sql`, so you can just pass this file to `psql`.

PS. `misc-unit-mulf_valid` is not the test of Pgfe. This is here for now since Pgfe is automatically synced from Cefeika project, but I'm on the way to clean up the Pgfe repo from the code on which the Pgfe is not actually depends.

Thanks for using Pgfe!

**Comment by dmitigr** [comment]
Fixed in 376eae448215b05171576c43e66468e3242e47f7
